src/python/ensembl/io/genomio/process_gff3.py

```diff
@@ -547,30 +541,30 @@ def cds_gene(self, cds: SeqFeature) -> SeqFeature:
         exon = SeqFeature(cds.location, type="exon")
         exon.qualifiers["source"] = cds.qualifiers["source"]
         transcript.sub_features.append(exon)
-        
+
         # Create a gene, add the transcript
         gene_type = "gene"
-        if("pseudo" in cds.qualifiers and cds.qualifiers["pseudo"][0] == "true"):
+        if "pseudo" in cds.qualifiers and cds.qualifiers["pseudo"][0] == "true":
```

Comment:
```suggestion
        if ("pseudo" in cds.qualifiers) and (cds.qualifiers["pseudo"][0] == "true"):
```
This is just a suggestion that I find more readable (and safer).